### ISSUE TITLE
class methods are incorrectly reported as anon

### ISSUE THREAD

**Issue by georgejecook** [rokucommunity/roku-debug#154]
![image](https://github.com/rokucommunity/roku-debug/assets/1214545/69936ada-3a3c-4ec6-bcbf-6285f6ff95d0)

given that we know the line of code, can we not use the source map to get the correct name in the stack?

**Comment by TwitchBronBron** [comment]
Yeah that's definitely a possibility. We just need someone with the bandwidth to implement it.
